Documentation/CRDs/Object-Storage/ceph-object-store-user-crd.md

```diff
@@ -36,6 +36,8 @@ spec:
 
 * `store`: The object store in which the user will be created. This matches the name of the objectstore CRD.
 * `displayName`: The display name which will be passed to the `radosgw-admin user create` command.
+* `clusterNamespace`: The namespace where the parent CephCluster and CephObjectStore are found. If not specified,
```

Comment:
By itself, this seems too open. The user could create the object user in any namespace, without any admin control about whether they should have access. What if the object store had a setting `allowUsersInAnyNamespace: true` that would enable this feature? This would effectively be the same as the OBC or COSI storage class that effectively gives access to other namespaces to request buckets.

Reply:
sounds perfect

Reply:
The setting `allowUsersInAnyNamespace` is now added as well